packages/desktop-client/src/components/reports/disabledList.ts

```diff
@@ -1,41 +1,43 @@
+import { t } from 'i18next';
+
 const intervalOptions = [
   {
-    description: 'Daily',
-    defaultRange: 'This month',
+    description: t('Daily'),
+    defaultRange: t('This month'),
```

Comment:
cc @carkom should these be translated?

Reply:
bump @carkom

Reply:
It's difficult to know. If you capture all of them then it should be fine. If it's translated then saved then all the options to reference are also translated then it should be fine as long as we capture them all.

However, if you save a report while the app is being translated and then switch back to English (or any other language) then the report will error out.

Reply:
Are any of these things stored in the DB? If tgey are then we definitely shouldn't be translating them.

Reply:
@a-gradina would you mind removing the translations from all the `defaultRange` values? I'm happy yo merge once we have that done :)

Sorry for this PR taking so long

Reply:
bump @a-gradina . I think there was a misunderstanding. You can keep the translations for `description`.  But remove from `defaultRange`.